VideoLocker/src/test/java/org/edx/mobile/profiles/UserProfilePresenterTest.java

```diff
@@ -89,6 +85,13 @@ public void whenInteractorEmitsProfileImageError_noInteractionsWithView() {
     }
 
     @Test
+    public void whenInteractorEmitsTabs_showsTabsOnView() {
```

Comment:
We want to follow AOSP guidelines here so I guess this would be something like

`testShowTabs_whenInteractorEmitsTabs`